pyqtgraph/parametertree/ParameterTree.py

```diff
@@ -30,9 +30,8 @@ def __init__(self, parent=None, showHeader=True):
         self.itemCollapsed.connect(self.itemCollapsedEvent)
         self.lastSel = None
         self.setRootIsDecorated(False)
-
+        self.setAlternatingRowColors(False)
         app = mkQApp()
```

Comment:
can be removed now

Reply:
oh, just noticed 10 lines up where that property is set to `True` ... whoops, yeah just remove those entries and that's better...

It's too bad on Windows the alternate color that is picked is the "accent" color on the windows theme, not something subtly different